src/Resources/Interfaces/QueriableResourceInterface.php

```diff
@@ -11,19 +13,43 @@
  */
 interface QueriableResourceInterface extends ResourceInterface
 {
+    const DEFAULT_PAGE = 1;
```

Comment:
Is there a reason to put the constants in the interface?

Reply:
I think it is good to know that pagination starts at 1 and not at 0 from the outside. In general I think about zero-based stuff. So I think it is good to communicate that things start at 1